test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.AttributesWithBooleanValues.html

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Not broken. The `Trim` convention used in tests hides the inconsistency with the output file.